### ISSUE TITLE
NetworkOperationsWebService class

### ISSUE THREAD

**Issue by jmfee-usgs** [usgs/earthquake-network-operations#5]
- constructor(StationTelemetryFactory)
- run(params)
- parseQuery(params)

**Comment by ehunter-usgs** [comment]
What should the output format look like?

```
{
  metadata: {
    station:
    network:
    error:
    url:
  },
  data: [ {station}, {station}, ... ]
}

**Comment by ehunter-usgs** [comment]
Each station object is a geojson feature as described in the readme
https://github.com/jmfee-usgs/earthquake-website/blob/e55aa86284c8d9c1edca1e01fa0b47d1d803f81f/src/htdocs/monitoring/operations/stations/README.md

**Comment by jmfee-usgs** [comment]
Merged #15, but needs more work:
- create entrypoint page (basically example/service.php but in htdocs)
- `format_station_geojson` should return null instead of just returning when its input is not truthy
- `run` should catch exceptions, and when there are errors set an appropriate http status code (400 for bad params, 500 for server errors) and output an error message in JSON format.
- include `updated`, `network_code`, and `station_code` as properties in output; format updated as iso8601 date
